src/main/js/main_html5.js

```diff
@@ -494,6 +497,9 @@ require("../../../html5-common/js/utils/environment.js");
           var mode = (!!params && params.mode) || 'showing';
 
           $(_video).append("<track class='" + TRACK_CLASS + "' kind='subtitles' label='" + label + "' src='" + src + "' srclang='" + language + "' default>");
+          if (_video.textTracks && _video.textTracks[0]) {
+            _video.textTracks[0].oncuechange = onClosedCaptionCueChange;
```

Comment:
this will never get published by multiple tracks at once, right?  Just the enabled one?